main.go

```diff
@@ -105,6 +107,10 @@ func setupFlags(cmd *cobra.Command) {
 	flagSet.BoolVar(&cfg.NetworkConfig.BridgeConfig.IPForward, "ipforward", true, "Enable ipforward")
 	flagSet.BoolVar(&cfg.NetworkConfig.BridgeConfig.UserlandProxy, "userland-proxy", false, "Enable userland proxy")
 
+	// log config
+	flagSet.StringVar(&cfg.DefaultLogConfig.LogDriver, "log-driver", "", "Set default log driver")
```

Comment:
I think we can set the `log-driver` to `jsonfile`, like mody does. WDTY?

Reply:
yep. If `log-driver` is empty, CI will fail.